pkg/agent/iptables/iptables.go

```diff
@@ -42,24 +56,34 @@ var (
 	// Antrea uses 10th for SNAT.
 	masqueradeBit   = uint(10)
 	masqueradeValue = 1 << masqueradeBit
-	masqueradeMark  = fmt.Sprintf("%#08x/%#08x", masqueradeValue, masqueradeValue)
+	masqueradeMark  = fmt.Sprintf("%#x/%#x", masqueradeValue, masqueradeValue)
+
+	// RtTblSelectorValue selects which route table to use to forward service traffic back to host gateway gw0.
```

Comment:
I think `RtTblSelectorValue` is not `which route table`? It should be the mark value by which service traffic is identified?

Reply:
I think either description is OK? It is true this mark identifies the service traffic, but the propose of that identification is to select the right route table for service traffic.

traffic --> service traffic (iptable set RtTblSelectorValue) ----> ip rule matches RtTblSelectorValue and select service route table.